### ISSUE TITLE
Request: Add Pick Detail to CSV files (Hero Details Data)

### ISSUE THREAD

**Issue by HenrikRaz** [ebshimizu/stats-of-the-storm#322]
I would happily pay a penny or two for someone to add an extra column to the CSV files containing when the hero was picked in draft from 1 to 10. 

If it is at all possible.

**Comment by ebshimizu** [comment]
heyo, it's been a real long time since i've looked, but i don't think that data is readily accessible. you can certainly compute it from the given data but even the draft order is reconstructed in the replay so you'd have to run through each draft mode replay and compute those stats manually. so i guess i'm saying it's possible, but i'm not sure i'll get back around to implementing it :/

**Comment by HenrikRaz** [comment]
Fair enough - but if the old flame ever strikes you and you're up for the task, just let me know and I'll throw some Patreon at you :)
